operations/ledgerbalances.md

```diff
@@ -0,0 +1,46 @@
+# Ledger balances
+
+## Get all ledger balances
+
+Returns opening & closing balances of all specified ledgers for each day in the specified interval.
+
+### Request
+
+`[PlatformAddress]/api/connector/v1/ledgerBalances/getAll`
```

Comment:
@marektresnak also I've noticed that the endpoint is actually hidden behind FF but putting it in the docs? 🤔 So should it be available for everyone right away?